### ISSUE TITLE
missing title attribute on encryption status icons

### ISSUE THREAD

**Issue by uhoreg** [vector-im/riot-web#3786]
the encryption status icon (locked, unlocked, warning) next to messages in encrypted rooms should have a title attribute, both for accessibility reasons, and to explain what it means when someone hovers over it

**Comment by lampholder** [comment]
Not urgent, but this would really be a very valuable optimisation.
